internal/controllers/clusterextension_controller.go

```diff
@@ -206,19 +206,24 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alp
 	installedBundle, err := r.InstalledBundleGetter.GetInstalledBundle(ctx, ext)
 	if err != nil {
 		setInstallStatus(ext, nil)
-		// TODO: use Installed=Unknown
-		setInstalledStatusConditionFailed(ext, err.Error())
-		setStatusProgressing(ext, err)
+		// The error is put into Progressing
+		setInstalledStatusConditionUnknown(ext, err.Error())
+		setStatusProgressing(ext, errors.New("retrying to get installed bundle"))
```

Comment:
Seems backwards based on the added comment?

Reply:
@joelanford wanted the retry status under Progressing

Reply:
See: https://github.com/operator-framework/operator-controller/pull/1379#discussion_r1805462712

Reply:
That's fine, but the comment seems to be stating the opposite. If this is the desired ordering, we should probably update the comment such that it reflects what the code actually says.

Reply:
Forgot to remove that comment...